realm/kotlin-extensions/src/androidTest/kotlin/io/realm/CoroutineTests.kt

```diff
@@ -28,19 +27,14 @@ class CoroutineTests {
     private lateinit var configuration: RealmConfiguration
     private lateinit var testDispatcher: TestCoroutineDispatcher
     private lateinit var testScope: TestCoroutineScope
-    private lateinit var realm: Realm
 
     @Before
     fun setUp() {
         testDispatcher = TestCoroutineDispatcher()
         testScope = TestCoroutineScope(testDispatcher)
         configuration = configFactory.createConfiguration()
-        realm = Realm.getInstance(configuration)
-    }
 
-    @After
-    fun tearDown() {
-        realm.close()
+        Realm.getInstance(configuration).close()
```

Comment:
This doesn't actually do anything. `Realm.getInstance()` will increment the reference count and then close it again. But any other underlying Realms will not be closed.

Reply:
I had to do this because I'm using a dynamic realm in one of the tests, so I would have a schema error if I don't have it.

Reply:
I had to do this because I'm using a dynamic realm in one of the tests, so I would have a schema error if I don't have it.

Reply:
OK, you're right this is probably something else. I will remove it.